python/requirements.txt

```diff
@@ -11,6 +11,6 @@ pycodestyle==2.5.0
 pycparser==2.19
 pyflakes==2.1.1
 pymongo==3.11.0
-pymongocrypt==1.1.0
+pymongocrypt==1.0.1
```

Comment:
```suggestion
```

Reply:
I think I need to put in a note about this. You still want 1.0.1 for AWS.

Reply:
Hm, I guess the additive approach in your second suggestion could work too.